From 7d66ca1081c0626f03aa8f419ca253e88a10018e Mon Sep 17 00:00:00 2001
From: Viorel Suman <viorel.suman@nxp.com>
Date: Mon, 17 Sep 2018 09:31:24 +0300
Subject: [PATCH] MLK-19573-1: ASoC: fsl: dsd: make fsl_get_pins_state inline

Make fsl_get_pins_state function inline.

Signed-off-by: Viorel Suman <viorel.suman@nxp.com>
(cherry picked from commit badcb97ebd8c0aae89f76e979bcc801be35c7400)
---
 sound/soc/fsl/fsl_dsd.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/sound/soc/fsl/fsl_dsd.h
+++ b/sound/soc/fsl/fsl_dsd.h
@@ -29,7 +29,7 @@ static bool fsl_is_dsd(struct snd_pcm_hw
 	}
 }
 
-static struct pinctrl_state *fsl_get_pins_state(struct pinctrl *pinctrl,
+static inline struct pinctrl_state *fsl_get_pins_state(struct pinctrl *pinctrl,
 	struct snd_pcm_hw_params *params, u32 bclk)
 {
 	struct pinctrl_state *state = 0;
